src/i18next.js

```diff
@@ -116,7 +116,8 @@ class I18n extends EventEmitter {
 
   /* eslint consistent-return: 0 */
   loadResources(callback = noop) {
-    if (!this.options.resources) {
+    if (!this.options.resources || (this.options.partialBundledLanguages &&
+      (!this.options.resources[this.languages[0]] || !this.options.resources[this.languages[0]][this.options.ns]))) {
```

Comment:
Why you do this check? `this.options.resources[this.languages[0]]`

a) it's not sufficient to check only for language at index 0
c) [this.options.ns] can is an array anyway
b) if i remember right the backend won't load that lng-ns again if already in the resource store: https://github.com/i18next/i18next/blob/master/src/BackendConnector.js#L44

Reply:
I'm checking the language at index 0 following you comments on a previous PR for that same feature: https://github.com/i18next/i18next/pull/1066

I checked for the this.options.ns to allow passing a partial language for only one of the namespace used by the application (where other namespace would need to be remotely loaded).

Do you mean that I could simply check for the flag and let the BackendConnector load or not the resources?

One tangent comment: it would be much easier to follow what is an array and such if typings were used. Did you consider moving to typescript?

Reply:
the load function meanwhile changed to be reusable for the forced reload.

so i believe just checking for the flag would be sufficient -> but you should test that to be sure.

regarding typescript...won't use it for this module itself...but the typedefinitions will / should be added for next upcoming major version: https://github.com/i18next/i18next/issues/1129

Reply:
Just tested, and it works as expected. Submitting a change very soon.

Reply:
Added a commit that looks just at the flag.